.circleci/config.yml

```diff
@@ -660,7 +702,7 @@ workflows:
           attach: true
           dir: cephcsi
           requires:
-            - build-ceph
+            - build-ceph-16
```

Comment:
Note: In this PR, we build ceph before cephcsi, the requires is needed.
It can be deleted after this is merged.